src/smart-components/group/group-table-helpers.js

```diff
@@ -9,6 +9,9 @@ import AppLink from '../../presentational-components/shared/AppLink';
 import { getDateFormat } from '../../helpers/shared/helpers';
 import pathnames from '../../utilities/pathnames';
 import messages from '../../Messages';
+import { sortable } from '@patternfly/react-table';
+import { CheckIcon, CloseIcon } from '@patternfly/react-icons';
```

Comment:
We already have import from `@patternfly/react-icons` present, can you combine them?